docs/nodes-and-miners/run-a-node.md

```diff
@@ -1,5 +1,5 @@
 ---
```

Comment:
same as the miner content - a lot of changes have been made to the docs in the repo itself, will hae to verify if this is still correct. 

i want to find a way for the docs to essentially "import" external docs we want to use

Reply:
Can confirm that the content shown here is incorrect. will have to sendyou an update or merge this to the docs repo and update before it's merged to master

Reply:
Similar to the above, leaving this for now per our conversation, with the plan to copy/paste after initial fixes and then find a better solution.